package.json

```diff
@@ -64,6 +64,7 @@
     "build": "react-scripts --openssl-legacy-provider build",
     "test": "react-scripts --openssl-legacy-provider test --maxWorkers=4",
     "eject": "react-scripts eject",
+    "generate-types": "npx openapi-typescript https://appserver.zenml.io/openapi.json -o schema.d.ts",
```

Comment:
You may need a custom script here, because of the transformer


```ts
import fs from "node:fs";
import openapiTS from "openapi-typescript";

// Logging function with emojis
function log(message, emoji) {
	console.log(`${emoji} ${message}`);
}

log("Script started.", "✨");

const output = await openapiTS("https://appserver.zenml.io/openapi.json", {
	exportType: true,
	transform: (schema) => {
		customTransformer(schema);
	}
});

log("Writing output to file...", "📝");
fs.writeFileSync("./src/types/core.ts", output);

log("Script completed successfully.", "✅");

/**
 *
 * @param {import("openapi-typescript").SchemaObject} schema
 * @returns {import("openapi-typescript").SchemaObject}
 */
export function customTransformer(schema) {
	if (schema.type === "object" && !schema.properties) {
		schema.additionalProperties = true;
	}
	return schema;
}
```

otherwise the python types of `dict: any` will resolve to typescript `never`

Reply:
fixed

Reply:
@Cahllagerfeld Fixed. Kindly check the changes.